cottoncandy/interfaces.py

```diff
@@ -212,6 +212,34 @@ def get_bucket_objects(self, **kwargs):
         response = request.all()
         return response
 
+    def get_bucket_size(self, limit=10000000, page_size=10000000):
+        '''Counts the size of all objects in the current bucket.
+
+        Returns
+        -------
+	int 
+            The byte count of all objects in the bucket.
+
+        Notes
+        -----
+        Because paging does not work properly, if there are more than
+        limit,page_size number of objects in the bucket, this function will
+        underestimate the total size. Check the printed number of objects for
+        suspicious round numbers.
+        TODO(anunez): Remove this note when the bug is fixed.
+        '''
+        # We do this to make sure that we are in a valid bucket.
+        assert exists_bucket(self.bucket_name)
+
+        total_bytes = 0
+        num_objects = 0
+        for obj in self.get_bucket_objects(limit=limit, page_size=page_size):
+            total_bytes += obj.size
+            num_objects += 1
+        print(str(total_bytes) + " bytes over " + str(num_objects) 
```

Comment:
make it more human readable by also printing the size in MB in parenthesis. e.g. "1024 bytes (1MB) ..."